service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -99,6 +104,48 @@ void setup() {
             when(feeService.getFeesDataForC2(WITHOUT_NOTICE)).thenReturn(buildFeesData(feeForC2WithoutNotice));
         }
 
+        @Test
+        void shouldRetryPaymentsApiWhen500IsThrown() {
```

Comment:
it would be good to have test that verify no exception is thrown when first request failed but retry was successful.

Reply:
done